matlab/test/arrow/array/tBooleanArray.m

```diff
@@ -216,5 +216,27 @@ function TestIsEqualFalse(tc)
             % Test supplying more than two arrays to isequal
             tc.verifyFalse(isequal(array1, array1, array3, array4, array5)); 
         end
+
+        function TestNumNulls(testCase)
```

Comment:
There are tests that use `testCase` and `tc`. Can we use `testCase` or `tc` in all tests?

Reply:
Good catch! I'll switch `testCase` to `tc`.